core/ui/TestCases/DefaultTemplate.tid

```diff
@@ -0,0 +1,22 @@
+title: $:/core/ui/testcases/DefaultTemplate
+
+\whitespace trim
```

Comment:
You used `\whitespace trim` but there is no indentation for improved readability. 

IMO it would be nice to have indented templates. 

There are many tiddlers with no indent.